activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/AuditProducerIT.java

```diff
@@ -630,10 +650,322 @@ public void testTwoSubProcesses() {
         });
 
         // Clean up
+        // Clean up
+        runtimeService.deleteProcessInstance(subProcessId1, "Clean up");
+        runtimeService.deleteProcessInstance(subProcessId2, "Clean up");
+
         runtimeService.deleteProcessInstance(processInstanceId, "Clean up");
 
     }
 
+    @Test
+    public void testEmbeddedSubProcesses() {
```

Comment:
@CTI777, let's create another test suite to cover embedded subprocesses (`EmbeddedSubProcessAuditIT`). This class is becoming too big.